lastFm-api/js/request.js

```diff
@@ -0,0 +1,23 @@
+// API Key 6c42a6265b88915f11e9bd1aad2d389d
+
+function Request() {
+  this.xhr = new XMLHttpRequest();
+  
+  this.xhr.onload = function () {
+    if (this.readyState === 4) {
+      if (this.status === 200) {
+        console.log(this.responseText);
+      } else {
+        console.error(this.statusText);
+      }
+    }
+  };
+  this.xhr.onerror = function () {
+    console.error(this.statusText);
+  };
+}
+
+Request.prototype.sendRequest = function () {
```

Comment:
I changed the `sendRequest()` function, so now it can work with other methods too.